python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -826,6 +826,9 @@ def get_settings(self, settings_key: str) -> Any:
             return self._settings.get(settings_key)
         return {}
 
+    def get_backfill_settings(self) -> Mapping[str, Any]:
+        return self.get_settings("backfills")
```

Comment:
you'll want to add "backfills" to the schema here: https://github.com/dagster-io/dagster/blob/0ae36df912c034530d80404ab99e843578687086/python_modules/dagster/dagster/_core/instance/config.py#L393

and include a test that verifies that you can load a dagster.yaml with that config set and it is propagated to the instacne settings correctly

here is an example of such a test for sensor settings: https://github.com/dagster-io/dagster/blob/0ae36df912c034530d80404ab99e843578687086/python_modules/dagster/dagster_tests/daemon_sensor_tests/test_sensor_run.py#L2911-L2915